crates/bevy_pbr/src/render/pbr_lighting.wgsl

```diff
@@ -176,6 +229,27 @@ fn fresnel(f0: vec3<f32>, LdotH: f32) -> vec3<f32> {
     return F_Schlick_vec(f0, f90, LdotH);
 }
 
+// Given distribution, visibility, and Fresnel term, calculates the final
+// specular light.
+//
+// Multiscattering approximation:
+// <https://google.github.io/filament/Filament.html#listing_energycompensationimpl>
+fn specular_multiscatter(
+    input: ptr<function, LightingInput>,
+    D: f32,
+    V: f32,
+    F: vec3<f32>,
+    specular_intensity: f32,
+) -> vec3<f32> {
+    // Unpack.
+    let F0 = (*input).F0_;
+    let F_ab = (*input).F_ab;
+
+    var Fr = (specular_intensity * D * V) * F;
+    Fr *= 1.0 + F0 * (1.0 / F_ab.x - 1.0);
```

Comment:
The linked Filament doc uses `dfg.y` instead `F_ab.x`. I assume `F_ab` is the same as `dfg`, but is the .x vs .y correct?

Reply:
Ah, I see you just copied that from somewhere else. I guess we shouldn't touch this in this PR then.

Reply:
Yeah, we might want to check this later but I just copy and pasted this one.